pinot-common/src/main/java/org/apache/pinot/common/config/IndexingConfig.java

```diff
@@ -56,9 +56,6 @@
   @UseChildKeyHandler(SimpleMapChildKeyHandler.class)
   private Map<String, String> _streamConfigs = new HashMap<>();
 
-  @ConfigKey("streamConsumptionConfig")
```

Comment:
Is this a backward incompatible change?

Reply:
This config is for the real-time replica-group, which is not supported yet.
After checking in this pr, real-time replica-group will be supported with the instance partitions.